src/utils/whitelistApis.ts

```diff
@@ -2782,6 +2782,14 @@ export const API_LIST = {
                 ROLE.PUBLIC,
               ],
           },
+          '/proxies/v8/content/v2/discard/:id': {
+            checksNeeded: [CHECK.ROLE],
+            // tslint:disable-next-line: object-literal-sort-keys
+            ROLE_CHECK: [
+                ROLE.CONTENT_CREATOR,
+                ROLE.EDITOR
```

Comment:
No need of this role.
Only content_creator is fine

Reply:
done